main.cpp

```diff
@@ -330,6 +337,8 @@ struct House
     int cleanHouse(int numDirtyRooms, int secondsPerRoom);
 
     Bathroom bathroomA;
+
+    void tFSGD();
```

Comment:
Name things what they are or what they do
do not use anacronyms in function names.
- [ ] revise